datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -47,14 +56,17 @@ export function filterSchemaRows(
         entityRegistry,
         formattedFilterText,
     );
+
     const finalFieldPaths = new Set();
     const expandedRowsFromFilter = new Set();
 
     rows.forEach((row) => {
         if (
             matchesFieldName(row.fieldPath, formattedFilterText) ||
             matchesEditableTagsOrTerms(row, filteredFieldPathsByEditableMetadata) ||
-            matchesTagsOrTerms(row, formattedFilterText, entityRegistry) // non-editable tags and terms
+            matchesEditableDescription(row, filteredFieldPathsByEditableMetadata) ||
+            matchesTagsOrTermsOrDescription(row, formattedFilterText, entityRegistry) || // non-editable tags and terms
+            matchesDescription(row.description, formattedFilterText) // non-editable description
```

Comment:
since we can rename `matchesEditableTagsOrTerms` to `matchesEditableTagsOrTermsOrDescription` with the new functionality change, we don't need `matchedEditableDescription` here. also, since you're calling `matchesTagsOrTermsOrDescription` for non-editable info, you don't need `matchesDescription` either.